x/cork/keeper/keeper.go

```diff
@@ -222,6 +303,56 @@ func (k Keeper) GetApprovedCorks(ctx sdk.Context, threshold sdk.Dec) (approvedCo
 	return winningCorks
 }
 
+func (k Keeper) GetApprovedScheduledCorks(ctx sdk.Context, currentBlockHeight uint64, threshold sdk.Dec) (approvedCorks []types.Cork) {
+
+	corksForBlockHeight := make(map[uint64][]types.Cork)
+	var corkPowersForBlockHeight map[uint64][]int64
```

Comment:
Should the accumulated powers here be a `uint64`?

Reply:
Made it so